_includes/wiki/it/footertext.md

```diff
@@ -0,0 +1,13 @@
+Scarica Jamulus](https://sourceforge.net/projects/llcon/files/){:.button}
```

Comment:
You need to add a new line and a [ before Scarica. It's something strange in the plugin.